WordPress/src/main/java/org/wordpress/android/push/GCMMessageService.java

```diff
@@ -461,27 +461,19 @@ private void buildAndShowNotificationFromNoteData(Context context, Bundle data,
             }
 
 
-            showNotification(context, pushId, wpcomNoteID,
-                    noteType, data.getString("icon"), title, message, dontPlaySound);
-        }
-
-        private void showNotification(Context context, int pushId, String wpcomNoteID, String noteType,
-                                      String largeIconUri, String title, String message,
-                                      boolean dontPlaySound) {
-
             // Build the new notification, add group to support wearable stacking
             NotificationCompat.Builder builder = getNotificationBuilder(context, title, message);
-
+            String largeIconUri = data.getString("icon");
```

Comment:
we could get rid of this local var and just include `data.getString("icon")` as the first parameter to `getLargeIconBitmap()`